src/main/java/htsjdk/samtools/cram/build/CompressionHeaderFactory.java

```diff
@@ -80,36 +77,36 @@ public CompressionHeader build(final List<CramCompressionRecord> records, Substi
 
         final CompressionHeaderBuilder builder = new CompressionHeaderBuilder(coordinateSorted);
 
-        builder.addExternalIntegerRansOrderZeroEncoding(DataSeries.AP_AlignmentPositionOffset);
-        builder.addExternalByteRansOrderOneEncoding(DataSeries.BA_Base);
+        builder.addExternalRansOrderZeroEncoding(DataSeries.AP_AlignmentPositionOffset);
```

Comment:
The typed versions (Byte, Integer, etc.) actually all did the same thing because their *encoding parameters* are identical.  The Codecs differ, but that's not relevant here.